docs/src/man/reshaping_and_pivoting.md

```diff
@@ -5,89 +5,77 @@ Reshape data from wide to long format using the `stack` function:
 julia> using DataFrames, CSV
 
-julia> iris = DataFrame(CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")));
-
-julia> first(iris, 6)
-6×5 DataFrame
-│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species     │
-│     │ Float64     │ Float64    │ Float64     │ Float64    │ String      │
-├─────┼─────────────┼────────────┼─────────────┼────────────┼─────────────┤
-│ 1   │ 5.1         │ 3.5        │ 1.4         │ 0.2        │ Iris-setosa │
-│ 2   │ 4.9         │ 3.0        │ 1.4         │ 0.2        │ Iris-setosa │
-│ 3   │ 4.7         │ 3.2        │ 1.3         │ 0.2        │ Iris-setosa │
-│ 4   │ 4.6         │ 3.1        │ 1.5         │ 0.2        │ Iris-setosa │
-│ 5   │ 5.0         │ 3.6        │ 1.4         │ 0.2        │ Iris-setosa │
-│ 6   │ 5.4         │ 3.9        │ 1.7         │ 0.4        │ Iris-setosa │
-
-julia> last(iris, 6)
-6×5 DataFrame
-│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species        │
-│     │ Float64     │ Float64    │ Float64     │ Float64    │ String         │
-├─────┼─────────────┼────────────┼─────────────┼────────────┼────────────────┤
-│ 1   │ 6.7         │ 3.3        │ 5.7         │ 2.5        │ Iris-virginica │
-│ 2   │ 6.7         │ 3.0        │ 5.2         │ 2.3        │ Iris-virginica │
-│ 3   │ 6.3         │ 2.5        │ 5.0         │ 1.9        │ Iris-virginica │
-│ 4   │ 6.5         │ 3.0        │ 5.2         │ 2.0        │ Iris-virginica │
-│ 5   │ 6.2         │ 3.4        │ 5.4         │ 2.3        │ Iris-virginica │
-│ 6   │ 5.9         │ 3.0        │ 5.1         │ 1.8        │ Iris-virginica │
-
-julia> d = stack(iris, 1:4);
-
-julia> first(d, 6)
-6×3 DataFrame
-│ Row │ Species     │ variable    │ value   │
-│     │ String      │ Cat…        │ Float64 │
-├─────┼─────────────┼─────────────┼─────────┤
-│ 1   │ Iris-setosa │ SepalLength │ 5.1     │
-│ 2   │ Iris-setosa │ SepalLength │ 4.9     │
-│ 3   │ Iris-setosa │ SepalLength │ 4.7     │
-│ 4   │ Iris-setosa │ SepalLength │ 4.6     │
-│ 5   │ Iris-setosa │ SepalLength │ 5.0     │
-│ 6   │ Iris-setosa │ SepalLength │ 5.4     │
-
-julia> last(d, 6)
-6×3 DataFrame
-│ Row │ Species        │ variable   │ value   │
-│     │ String         │ Cat…       │ Float64 │
-├─────┼────────────────┼────────────┼─────────┤
-│ 1   │ Iris-virginica │ PetalWidth │ 2.5     │
-│ 2   │ Iris-virginica │ PetalWidth │ 2.3     │
-│ 3   │ Iris-virginica │ PetalWidth │ 1.9     │
-│ 4   │ Iris-virginica │ PetalWidth │ 2.0     │
-│ 5   │ Iris-virginica │ PetalWidth │ 2.3     │
-│ 6   │ Iris-virginica │ PetalWidth │ 1.8     │
+julia> iris = DataFrame(CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")))
+150×5 DataFrame
+ Row │ SepalLength  SepalWidth  PetalLength  PetalWidth  Species
+     │ Float64      Float64     Float64      Float64     String
+─────┼──────────────────────────────────────────────────────────────────
+   1 │ 5.1          3.5         1.4          0.2         Iris-setosa
```

Comment:
@ronisbr - @nalimilan that floating point columns seem aligned to the left not to the right. Why it is so?

Reply:
Hum, this is because the algorithm to align the decimal points. In this case, I need them to be aligned to the left, otherwise it will not work.

Reply:
OK - so we leave it as is.

Reply:
@bkamins I am not entirely sure, but maybe I can fix this (my concern is adding a lot of processing). Can I try in another PR?

Reply:
That would be nice. Since we're changing printing, which will force dependent packages to update their doctests, better break everything at the same time in 0.22.

Reply:
@ronisbr - so please check it and let us know. From my perspective this is not a critical thing to have. We are aligning floats to `.` anyway.

The point of right alignment of integers is that they can be easily compared, but this goal is already met with `.` alignment.

Reply:
Maybe the only question is how numbers in scientific notation are aligned? (and do we consistently make sure that all is scientific or non-scientific notation)

Have you tested this?

Reply:
> @ronisbr - so please check it and let us know. From my perspective this is not a critical thing to have. We are aligning floats to `.` anyway.

OK, I will do some tests.

> Maybe the only question is how numbers in scientific notation are aligned? (and do we consistently make sure that all is scientific or non-scientific notation)
> 
> Have you tested this?

We are using `print` to convert the number. Hence, if the number has order less than -5 or high than +5, then it will be printed in scientific notation. This is taken care and everything is aligned to the decimal point.

Reply:
> This is taken care and everything is aligned to the decimal point.

The same is done in Base, so I guess it is OK.